resources/hvac.rb

```diff
@@ -4112,13 +4221,28 @@ def self.get_unitary_system_from_air_loop_hvac(air_loop)
     return nil
   end
 
+  def self.get_evap_cooler_from_air_loop_hvac(air_loop)
+    # Returns the evap cooler or nil
+    air_loop.supplyComponents.each do |comp|
+      next unless comp.to_EvaporativeCoolerDirectResearchSpecial.is_initialized
+
+      return comp.to_EvaporativeCoolerDirectResearchSpecial.get
+    end
+    return nil
+  end
+
   def self.get_unitary_system_air_loops(model, runner, thermal_zone)
     # Returns the unitary system(s), cooling coil(s), heating coil(s), and air loops(s) if available
     unitary_system_air_loops = []
     thermal_zone.airLoopHVACs.each do |air_loop|
       system = get_unitary_system_from_air_loop_hvac(air_loop)
       next if system.nil?
 
+      # skip evap cooler dummy unitary system
+      if system.additionalProperties.getFeatureAsString(Constants.SizingInfoHVACCoolType).is_initialized
+        next if system.additionalProperties.getFeatureAsString(Constants.SizingInfoHVACCoolType).get == Constants.ObjectNameEvaporativeCooler
```

Comment:
This filtered out evap cooler dummy unitary system, which blocked sizing this system in hvac_sizing.rb.